PulseSampleApp/app/src/main/java/com/ooyala/sample/players/PulsePlayerActivity.java

```diff
@@ -29,180 +23,84 @@
-    }
-    // Automation Hook: to write Notifications to a temporary file on the device/emulator
-    String text = "Notification Received: " + argument + " - state: " + player.getState();
-
-    // Automation Hook: Write the event text along with event count to log file in sdcard if the log file exists
-    playbackLog.writeToSdcardLog(text);
-
-    Log.d(LOG_TAG, "Notification Received: " + argument + " - state: " + player.getState());
-  }
-
-  /*
-    Extras
-   */
-
-  /**
-   * Create a VideoItem from the bundled information send to this activity.
-   * @return The created {@link VideoItem}.
-   */
-  public VideoItem getVideoItem() {
-    VideoItem videoItem = new VideoItem();
-
-    videoItem.setTags(getIntent().getExtras().getStringArray("contentMetadataTags"));
-    videoItem.setMidrollPositions(getIntent().getExtras().getFloatArray("midrollPositions"));
-    videoItem.setContentTitle(getIntent().getExtras().getString("contentTitle"));
-    videoItem.setContentId(getIntent().getExtras().getString("contentId"));
-    videoItem.setCategory(getIntent().getExtras().getString("category"));
-    videoItem.setContentCode(getIntent().getExtras().getString("embedCode"));
-
-    return videoItem;
-  }
-
-  /** Start DefaultHardwareBackBtnHandler **/
-  @Override
-  public void invokeDefaultOnBackPressed() {
-    super.onBackPressed();
-  }
-
-  @Override
-  public void onActivityResult(int requestCode, int resultCode, Intent data){
-      if((resultCode == RESULT_OK || resultCode == RESULT_CANCELED) && requestCode == CLICK_THROUGH_REQUEST){
-        Log.i("DemoIntegration","Came back from clickthrough");
-        onResume();
-      }
-  }
+public class PulsePlayerActivity extends AbstractHookActivity {
+
+	public final static String getName() {
+		return "Pulse Player";
+	}
+
+
+	/**
+	 * Called when the activity is first created.
+	 */
+	@Override
+	public void onCreate(Bundle savedInstanceState) {
+		super.onCreate(savedInstanceState);
+		setTitle(getName());
+		setContentView(R.layout.player_skin_simple_layout);
+		completePlayerSetup(asked);
+	}
+
+	@Override
+	void completePlayerSetup(boolean asked) {
+		final VideoItem videoItem = getVideoItem();
+		// Get the SkinLayout from our layout xml
+		OoyalaSkinLayout skinLayout = (OoyalaSkinLayout)findViewById(R.id.ooyalaSkin);
+		// Create the OoyalaPlayer, with some built-in UI disabled
+		PlayerDomain domain = new PlayerDomain(DOMAIN);
+		Options options = new Options.Builder().setShowPromoImage(false).setUseExoPlayer(true).build();
+
+		player = new OoyalaPlayer(PCODE, domain, options);
+
+		//Create the SkinOptions, and setup the LayoutController
+		SkinOptions skinOptions = new SkinOptions.Builder().build();
+		playerSkinLayoutController = new OoyalaSkinLayoutController(getApplication(), skinLayout, player, skinOptions);
+		playerSkinLayoutController.addObserver(this);
+		player.addObserver(this);
+
+
```

Comment:
Remove this big spacing, if you want to have spacing it's ok but just add 1 line. In this case there's 2 lines spacing.